skel/handlers/index.go

```diff
@@ -26,8 +29,16 @@ func Setup(app *sonic.Runtime) {
 
 // index render with master layer
 func index(ctx *gin.Context) {
+	fullPath, _ := os.Getwd()
```

Comment:
Please always handle errors, if the function can return an error you can just return it.
Sometime we can just panic, for example when we have fatal errors during the initialization of the application.